uploadNeuroDB/minc_insertion.pl

```diff
@@ -502,6 +574,15 @@
 ################################################################
 exit 0;
 
+
+=pod
+
+=head3 logHeader()
```

Comment:
notice the same function in the tarchiveLoader pull request here:
https://github.com/aces/Loris-MRI/pull/246/files

I added a consistency item to the redmine ticket. I think we should follow what you have HERE and not have inputs to the function, but we need to make it consistently

Reply:
@MounaSafiHarab The problem is that in here, no variables are passed to the function, while in tarchiveLoader, there are 3 arguments directly passed to the function. I agree that we should make the logHeader() functions consistent across scripts but this should be a different ticket than documentation. 
Actually, I end up creating the same function for other scripts so maybe there is a way to have that function only once in one of the library instead of duplicating it across all the scripts. 
What do you think?

Reply:
yes, definitely a separate pull request, but why I bring it as part of the documentation stuff, is because in tarchiveLoader, when you run it now (after it has been `perlified`!), the logHeader function gives a warning (I noticed it the other day) and it has to do with passing the inputs to logHeader. I know you tested it but it is easy to miss this message on the screen as everything progresses normally afterwards... so this should be addressed before we release the documentation branch.
but by all means, the same function used in many scripts should being centralized, if possible, should be another ticket.

Reply:
@cmadjar 
I just verified, I do not think that the logHeader() function in tarchiveLoader is used any differently (i.e. I do not see any arguments passed directly).